bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +110,35 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        colon_pos = shortlog.find(':')
```

Comment:
And btw I want an option for enabling/disabling this "imperative" check, sometimes there could be false-positives (seems you have some) and then one would like maybe to disable this feature. By default it can be enabled I think^^